### PR TITLE
Correct name

### PR DESCRIPTION
**Pull request by x3ro**
🎉

### CODE REVIEW

File: draft-ietf-webpush-protocol.xml [hunk 1]
```diff
@@ -1500,7 +1500,7 @@ HEADERS      [stream 82] +END_STREAM
     <section title="Acknowledgements">
       <t>
         Significant technical input to this document has been provided by Ben
-        Bangert, Peter Beverloo, Kit Cambridge, JR Conlin, Lucas Jenns, Matthew Kaufman,
```
Comment (martinthomson) [inline draft-ietf-webpush-protocol.xml:1503]: Yeah, that's a problem.  We can't (yet) use non-ascii for these documents (which is super annoying).  What is your preferred ASCII form Jens or Jenss perhaps?

Comment (x3ro) [inline draft-ietf-webpush-protocol.xml:1503]: Sorry, I was unaware of that limitation. I've modified the PR to use the ASCII-friendly spelling of my name 😄

Comment (martinthomson) [inline draft-ietf-webpush-protocol.xml:1503]: Frankly, I think that the situation stinks.  And we do have plans to fix it, but those plans take time.

**Comment by brianraymor** [comment]
@x3ro  - Apologies for the mis-spelling. I was focused on using a "ss" for the reasons that Martin outlined and managed to use a "nn" instead. I'm waiting to see if there are further comments from the chairs and/or area director before submitting webpush-09.

**Comment by x3ro** [comment]
@brianraymor No worries :)

**Pull request merged**